packages/nextjs-mf/src/plugins/container/InvertedContainerRuntimeModule.ts

```diff
@@ -1,139 +1,69 @@
 import { normalizeWebpackPath } from '@module-federation/sdk/normalize-webpack-path';
 import type { Module } from 'webpack';
+import ContainerEntryModule from '@module-federation/enhanced/src/lib/container/ContainerEntryModule';
 
 const { RuntimeModule, Template, RuntimeGlobals } = require(
   normalizeWebpackPath('webpack'),
 ) as typeof import('webpack');
+
 interface InvertedContainerRuntimeModuleOptions {
-  runtime: string;
-  remotes: Record<string, string>;
   name?: string;
-  debug?: boolean;
-  container?: string;
 }
 
 class InvertedContainerRuntimeModule extends RuntimeModule {
   private options: InvertedContainerRuntimeModuleOptions;
 
   constructor(options: InvertedContainerRuntimeModuleOptions) {
-    super('inverted container startup', RuntimeModule.STAGE_ATTACH + 3);
+    super('inverted container startup', RuntimeModule.STAGE_TRIGGER);
```

Comment:
runtimeModule goes at bottom of webpack runtime, the final step of "host" runtime chunk is to (globalScope[name] || moduleCache) = require(container/entry/host_own_remote_entry_module); @2heal1 the same could be done to /runtime. So you do not need to prepend entrypoints with runtime initialization. We could addDependency like we do to remote, then in RuntimeModule, we require__(federationRuntimeEntry) during bootup.